components/Init/src/virtio_irq.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022, UNSW (ABN 57 195 873 179)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+/**
+ * Hardcoded virtual IRQ numbers for virtual devices. These have to be unique
+ * for each device.
+ *
+ * @todo: Having hardcoded numbers might become an issue as the list of virtual
+ * devices grows. Might be worth it to have an IRQ number allocator.
+ */
+#define VIRTIO_NET_IRQ              6
+#define VIRTIO_BLK_IRQ              7
+
+/* Number 8 is occupied by the RTC timer, do not use! */
+#define RTC_IRQ                     8
```

Comment:
It seems a bit odd to define `RTC_IRQ` here given the comment. Is there any code that really needs this define and expects it to come from this file?

Reply:
Thanks for pointing that out, did some clean-up for the rtc file(`mc146818rtc.c`).